app/components/courses/international_students_component.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="govuk-!-margin-bottom-8">
+  <h2 class="govuk-heading-l" id="section-international-students">International students</h2>
+  <div data-qa="course__international_students">
+    <p class="govuk-body">We <%= visa_sponsorship_status %>.</p>
```

Comment:
Suggestion: move the "we" and the full stop into the conditional text (even if they all currently use this).